browser/main/Main.js

```diff
@@ -152,24 +153,12 @@ class Main extends React.Component {
       document.body.setAttribute('data-theme', 'default')
     }
 
-    const supportedLanguages = [
-      'sq',
-      'zh-CN',
-      'zh-TW',
-      'da',
-      'fr',
-      'de',
-      'hu',
-      'ja',
-      'ko',
-      'no',
-      'pl',
-      'pt',
-      'ru',
-      'es-ES'
-    ]
+    let locales = languages.reduce(function (localeList, locale) {
```

Comment:
please confirm [this review](https://github.com/BoostIO/Boostnote/pull/1818#discussion_r183685530)